app/models/balance.rb

```diff
@@ -0,0 +1,87 @@
+# This describes disk quota utilization for a given user and volume
```

Comment:
Update comment to describe balance instead of disk quota. Maybe in parentheses (RUs) in comments for us on Gateways?

Reply:
I would prefer that the comment refer to `config.unit` specifically, and call out resource units as what are used at OSC.

Reply:
Shouldn't that go in README or does it belong in model header too?